hello-slack/app.py

```
@@ -0,0 +1,378 @@
+            # 処理中メッセージ
+            say(f"<@{user}> 音声ファイルを受け付けました。文字起こしと話者分離を開始します...", thread_ts=thread_ts)
+
+            # 3. WAVファイルへの変換 (Resemblyzer用)
+            wav_file_path = os.path.join(temp_dir, "temp_audio_for_diarization.wav")
+            logging.info(f"Converting to WAV: {original_file_path} -> {wav_file_path}")
+            try:
+                 audio = AudioSegment.from_file(original_file_path) # pydub で読み込み
+                 audio = audio.set_frame_rate(16000).set_channels(1)
+                 audio.export(wav_file_path, format="wav")
+                 logging.info("Conversion to WAV successful.")
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)